python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -159,6 +166,11 @@ def resolve_asset(self, _graphene_info: ResolveInfo):
             materialization_loader=self._latest_materialization_loader,
         )
 
+    def resolve_partitionMapping(self, _graphene_info: ResolveInfo):
```

Comment:
I know we haven't been super rigorous about this in the past, but I think it's worth adding a return type annotation here.